storage/storage.c

```diff
@@ -206,8 +204,8 @@ static secbool secequal(const void *ptr1, const void *ptr2, size_t n) {
 
 static secbool secequal32(const void *ptr1, const void *ptr2, size_t n) {
   assert(n % sizeof(uint32_t) == 0);
-  assert((uintptr_t)ptr1 % sizeof(uint32_t) == 0);
-  assert((uintptr_t)ptr2 % sizeof(uint32_t) == 0);
+  //  assert((uintptr_t)ptr1 % sizeof(uint32_t) == 0);
+  //  assert((uintptr_t)ptr2 % sizeof(uint32_t) == 0);
```

Comment:
Why the assertions were commented out?

Reply:
Restored https://github.com/trezor/trezor-firmware/commit/1afca92292fed9de3476bd0ba90ee1c0fc1600b2

(it was commented out because one of the previous iteration of the blockwise norcow had valid flag (one byte flag) at the beginning of the data, thus the data itself were not 32 bit aligned. Note that MCUs we currently use can handle unaligned access so the check is not strictly necessary)